cds_ils/importer/documents/api.py

```diff
@@ -85,14 +85,14 @@ def search_document_by_title_authors(title, authors, subtitle=None):
     document_search = current_app_ils.document_search_cls()
     if subtitle:
         search = (
-            document_search.query("match", title=title)
+            document_search.query("query_string", query=f"title:\"{title}\"")
```

Comment:
Interesting change! Can you maybe quickly explain why from `match` to `query_string`?